ds-collector-tests/cluster-one-node-vanilla-ssh-docker.make

```diff
@@ -0,0 +1,39 @@
+
+all: setup test teardown
+
+
+test:
+	# ds-collector over SSH
+	docker exec -t ds-collector-tests_bastion_1 /collector/ds-collector -T -f /ds-collector-tests/test-collector-ssh.conf -n ds-collector-tests_cassandra-00_1
+	docker exec -t ds-collector-tests_bastion_1 /collector/ds-collector -T -p -f /ds-collector-tests/test-collector-ssh.conf -n ds-collector-tests_cassandra-00_1
+	docker exec -t ds-collector-tests_bastion_1 /collector/ds-collector -X -f /ds-collector-tests/test-collector-ssh.conf -n ds-collector-tests_cassandra-00_1
+	# test archives exist
+	if ! ( docker exec ds-collector-tests_bastion_1 ls /tmp/datastax/ ) | grep -q ".tar.gz" ; then echo "Failed to generate artefacts in the SSH cluster" ; ( docker exec ds-collector-tests_bastion_1 ls /tmp/datastax/ ) ; exit 1 ; fi
+	# ds-collector over docker
+	./collector/ds-collector -T -f test-collector-docker.conf -n ds-collector-tests_cassandra-00_1
+	./collector/ds-collector -T -p -f test-collector-docker.conf -n ds-collector-tests_cassandra-00_1
```

Comment:
as discussed, the ping can fail and at this stage we do not know if it fails because of a script issue, or fails because the ping itself fails. the return code is non-zero but it does not exit.